internal/shim-sev/src/syscall.rs

```diff
@@ -196,10 +199,10 @@ pub fn syscall_rust(
         // Enarx syscalls
 
         // Fixme
-        let ret: sallyport::Result = Err(libc::ENOSYS);
+        let ret: sallyport::Result = Ok([0.into(), SEV_TECH.into()]);
 
         match ret {
-            Err(e) => (e.checked_neg().unwrap() as usize, orig_rdx),
```

Comment:
If there is only one return value it must return orig_rdx in the second.

Reply:
Got it, I put it back the way it was.